apps/remote_control/lib/lexical/remote_control/search/store/mnesia/schema.ex

```diff
@@ -0,0 +1,171 @@
+defmodule Lexical.RemoteControl.Search.Store.Mnesia.Schema do
+  alias Lexical.Project
+  alias Lexical.RemoteControl.Search.Indexer.Entry
+  alias Lexical.RemoteControl.Search.Store.Mnesia
+  alias Lexical.RemoteControl.Search.Store.Mnesia.State
+  alias Lexical.VM.Versions
+
+  import Record
+
+  @fields [:path, :reference, :value]
```

Comment:
What do you think about renaming `:reference` to `:ref` to avoid any potential confusion between this and entry subtypes, which can be `:definition | :reference`?

```suggestion
  @fields [:path, :ref, :value]
```